litmus-portal/frontend/src/pages/MyHub/index.tsx

```diff
@@ -70,47 +85,86 @@ const MyHub = () => {
-                              >
-                                {hub.HubName}
-                              </Typography>
-                              <Typography
-                                variant="h6"
-                                align="center"
-                                className={classes.hubBranch}
-                              >
-                                {hub.GitURL.split('/')[4]}/{hub.GitBranch}
-                              </Typography>
-                            </CardContent>
-                          </Card>
-                        );
-                      })}
+                      {totalHubs &&
+                        totalHubs.map((hub: HubDetails) => {
+                          return (
+                            <Paper
+                              key={hub.id}
+                              elevation={3}
+                              className={classes.cardDivChart}
+                            >
+                              <CardContent className={classes.cardContent}>
+                                <Typography
+                                  className={
+                                    hub.IsAvailable
+                                      ? classes.connected
+                                      : classes.error
+                                  }
+                                >
+                                  {hub.IsAvailable ? 'Connected' : 'Error'}
+                                </Typography>
+                                <img
+                                  src="/icons/my-hub-charts.svg"
+                                  alt="add-hub"
+                                />
+                                <Typography
+                                  variant="h6"
+                                  align="center"
+                                  className={classes.hubName}
+                                >
+                                  {hub.HubName}
+                                </Typography>
+                                <Typography
+                                  variant="h6"
+                                  align="center"
+                                  className={classes.hubBranch}
+                                >
+                                  {hub.RepoURL.split('/')[4]}/{hub.RepoBranch}
+                                </Typography>
+                                <Typography className={classes.totalExp}>
+                                  {parseInt(hub.TotalExp, 10) > 0
+                                    ? `${hub.TotalExp} experiments`
+                                    : '[Error: could not connect]'}
+                                </Typography>
+                                <hr className={classes.horizontalLine} />
+                                {hub.IsAvailable ? (
+                                  <ButtonFilled
+                                    styles={{ width: '100%' }}
```

Comment:
place all inline styling in `styles.ts` file

Reply:
@ishangupta-ds The styles here is a prop that the ButtonFilled component takes, this can't be added in styles.ts file